server/apps/api/management/commands/update_ip_data.py

```diff
@@ -1,12 +1,102 @@
 # -*- coding: utf-8 -*-
 
+import json
+import hashlib
+from time import sleep
+
+import requests
+
 from django.core.management.base import BaseCommand
+from django.db.models import Count
+
+from server.apps.api.models import Case, Domain
+from server.apps.api.logic import notifier
 
-from server.apps.api.logic.ips_data import update_ip_data
-from server.apps.api.logic.blocked import check_blocked
+
+MIN_IP_COUNT = 2
 
 
 class Command(BaseCommand):
     def handle(self, *args, **options):
         update_ip_data()
         check_blocked()
+
+
+def alert_to_slack(domain_names):
+    message = 'Blocked: {}'.format(', '.join(domain_names))
+    notifier.slack_message(
+        message=message,
+        channel='#tests',
+    )
+
+
+def blocked_domains():
+    domain_pks = (Case.objects
+                  .values('domain_id', 'client_hash')
+                  .distinct()
+                  .values('domain_id')
+                  .annotate(hash_count=Count('client_hash'))
+                  .values('domain_id', 'hash_count')
+                  .filter(hash_count__gte=MIN_IP_COUNT)
+                  .values_list('domain', flat=True))
+    return (Domain.objects
+            .filter(pk__in=domain_pks)
+            .values_list('domain', flat=True))
+
+
+def check_blocked():
+    domains = blocked_domains()
+    if not domains:
+        return
+    alert_to_slack(domains)
```

Comment:
This will notify us always, even about old records. Use timestamps to make it smarter.